src/System.Text.Json/src/System/Text/Json/Serialization/JsonClassInfo.AddProperty.cs

```diff
@@ -174,7 +174,7 @@ internal JsonPropertyInfo CreateRootObject(JsonSerializerOptions options)
 
         internal JsonPropertyInfo CreatePolymorphicProperty(JsonPropertyInfo property, Type runtimePropertyType, JsonSerializerOptions options)
         {
-            JsonPropertyInfo runtimeProperty = CreateProperty(property.DeclaredPropertyType, runtimePropertyType, property.ImplementedPropertyType, property?.PropertyInfo, Type, options);
+            JsonPropertyInfo runtimeProperty = CreateProperty(property.DeclaredPropertyType, runtimePropertyType, property.ImplementedPropertyType, property.PropertyInfo, Type, options);
```

Comment:
Looks like no tests cover the other branch either (although that could just be due to a test gap).
![image](https://user-images.githubusercontent.com/6527137/63068086-cb974000-bec6-11e9-8cda-5156734250a2.png)

Reply:
> Looks like no tests cover the other branch either (although that could just be due to a test gap).

There's no way to cover it.  If propertyInfo is null, the previous `property.DeclaredPropertyType` access will null ref.

Reply:
Right, of course. I was just noting that even though test coverage doesn't show coverage, it doesn't necessarily mean the null check is unnecessary (just looking at the branch coverage). It's clear the null check is redundant if you reason about the code.

Reply:
Ah, ok.